test/gtest/uct/test_event.cc

```diff
@@ -21,22 +19,41 @@ class test_uct_event : public uct_test {
         m_e1 = uct_test::create_entity(0);
         m_entities.push_back(m_e1);
 
-        check_skip_test();
+        m_async_event_ctx.wakeup_fd.fd = -1;
```

Comment:
is't it already initialized in constructor->init?

Reply:
good catch, removed